src/firebaseWrapper.js

```diff
@@ -56,7 +56,51 @@ const listenForMessages = (conversationId, done) => {
 /**
  * Post Functions
  */
-// const sendPost = (userId, text, photo) => {};
+const sendPost = (userId, text, likes) => {
```

Comment:
could you make the interface for this function `const sendPost = (userId, text, photo, done) => {}`? Also combine this with `sendPostWithPhoto`. You can form the post object in the component. Seems like `likes` should always be zero when you're sending a post